src/design-tokens/mixins.css

```diff
@@ -29,26 +24,13 @@
   font-size: inherit;
   font-weight: var(--eds-font-weight-medium);
   line-height: inherit;
-  text-align: left;
-  display: inline-flex; /* 1 */
-  gap: 0.25em; /* 2 */
```

Comment:
This doesn't do anything without `display: inline-flex` so I moved this to the `ClickableStyle` styles where `display: inline-flex` is still present.